### DISCUSSION TITLE
How to add refresh button to each component in react

### DISCUSSION THREAD

**Discussion by zzy-life** [ant-design/ant-design#48912] [Q&A]
For example, for such a demand, is there any better way? Refreshing the current component is also possible.
![Screenshot of a comment on a GitHub issue showing an image, added in the Markdown, of an Octocat smiling and raising a tentacle.](https://i.imgur.com/2n8dYos.png)

Components are all popped up using antd's Modal. Just brush the Modal:
![Screenshot of a comment on a GitHub issue showing an image, added in the Markdown, of an Octocat smiling and raising a tentacle.](https://i.imgur.com/3vP68Hr.png)

**Comment by zzy-life** [comment]
For example, if you use iframe to implement Modal, you can use reload to refresh the browser (Figure 1 is how it is implemented)
However, if reload is used in the component form, the url will be refreshed, and the requirement only requires refreshing the Modal.

**Comment by zzy-life** [comment]
After the modal pops up, the data will change (other users have manipulated this data), so the user needs a refresh button to refresh the data in the current modal.

**Comment by MadCcc** [comment]
推荐使用 https://swr.vercel.app/zh-CN

**Reply by zzy-life** [reply]
好像要修改每个页面的请求方式...，工作量太大了。
最理想的状态就是修改Modal源码，或者封装个组件？
